lib/MultiCompiler.js

```diff
@@ -11,7 +11,18 @@ const ConcurrentCompilationError = require("./ConcurrentCompilationError");
 const MultiStats = require("./MultiStats");
 const MultiWatching = require("./MultiWatching");
 
+/** @typedef {import("./Compiler")} Compiler */
+
+/** @typedef {number} CompilerStatus */
```

Comment:
I had to use `number` instead of `0|1|2` to avoid extra type annotations `watch`